src/views/ont/ONTPoolIndex.vue

```diff
@@ -127,6 +128,9 @@ export default {
   },
   computed: {
     ...mapGetters('traction/ont/pools', ['pools']),
+    printingStore() {
+      return usePrintingStore()
```

Comment:
I think the better place to define usePrintingStore() is in the setup function. This also makes it consistent with how useQueryParams() is called.